File: compose-app/shared/src/commonMain/kotlin/com/sales/app/domain/model/Item.kt
package com.sales.app.domain.model

data class Item(
    val id: Int,
    val name: String,
    val altName: String?,
    val brand: String?,
    val size: String?,
    val uqc: Int,
    val hsn: Int?,
    val accountId: Int,
    val taxId: Int? = null
)

data class Uqc(
    val id: Int,
    val uqc: String,
    val quantity: String?,
    val type: String?,
    val active: Boolean
)

data class Tax(
    val id: Int,
    val schemeName: String,
    val country: String? = null,
    val tax1Name: String?,
    val tax1Val: Double?,
    val tax2Name: String?,
    val tax2Val: Double?,
    val tax3Name: String?,
    val tax3Val: Double?,
    val tax4Name: String?,
    val tax4Val: Double?,
    val active: Boolean
)

File: backend/app/Models/Item.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Item extends Model
{
    use SoftDeletes, \App\Traits\HasLog, \Illuminate\Database\Eloquent\Factories\HasFactory;

    public $timestamps = true;

    protected $fillable = [
        'name',
        'alt_name',
        'brand',
        'size',
        'uqc',
        'hsn',
        'account_id',
        'log_id',
        'tax_id',
    ];

    public function tax()
    {
        return $this->belongsTo(Tax::class);
    }

    public function account()
    {
        return $this->belongsTo(Account::class);
    }

    public function stock()
    {
        return $this->hasOne(Stock::class);
    }

    public function quoteItems()
    {
        return $this->hasMany(QuoteItem::class);
    }

    public function orderItems()
    {
        return $this->hasMany(OrderItem::class);
    }

    public function saleItems()
    {
        return $this->hasMany(SaleItem::class);
    }

    public function purchaseItems()
    {
        return $this->hasMany(PurchaseItem::class);
    }

    public function uqc_data()
    {
        return $this->belongsTo(Uqc::class, 'uqc', 'uqc');
    }
}
